backend/cmd/marcel.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/Zenika/MARCEL/backend/app"
+	"github.com/Zenika/MARCEL/backend/config"
+)
+
+var (
+	port uint
```

Comment:
Why not every thing in `Global` config structure ?

Reply:
Right, I'll move it.